Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
```

Comment:
Ah, good point. Cool. My thought with the symlink is to handle the cases where the location hasn't been updated yet (e.g. scripts or a copied `plist` rather than symlinked). Additionally, let's use the `PG_VERSION` file already in `var` to determine what the new folder name should be (rather than assuming it's 9.4).